### PR TITLE
FLOW-402 - Fix page sizing

### PR DESCRIPTION
**Pull request by dshoulders**

### CODE REVIEW

File: js/components/items-container.tsx [hunk 1]
```diff
@@ -365,18 +364,8 @@ class ItemsContainer extends React.Component<IComponentProps, IItemsContainerSta
...
-                    manywho.settings.flow(
-                        `paging.${model.componentType.toLowerCase()}`, this.props.flowKey,
-                    ) || 10,
-                    10,
-                );
-
-                if (!isNaN(paginationSize)) {
-                    limit = paginationSize;
-                }
+                limit = this.getPageSize(model, this.props.flowKey);
```
Comment (TomFullalove) [inline js/components/items-container.tsx]: Could limit just be a constant and assigned immediately where it's currently defined?

File: js/components/items-container.tsx [hunk 2]
```diff
@@ -386,12 +375,8 @@ class ItemsContainer extends React.Component<IComponentProps, IItemsContainerSta
 
         } else if (model.objectDataRequest || model.fileDataRequest) {
             objectData = model.objectData;
-            limit = parseInt(
-                manywho.settings.flow(
```
Comment (TomFullalove) [inline js/components/items-container.tsx]: It didn't use to apply pagination attributes here, so I want to be super sure that we should.
Looking at the docs https://github.com/manywho/ui-html5/wiki/Table-Component seems to suggest the pagination attribute only applies to List Values, but then the paginationSize is hinted to also apply to Service ObjectDataRequests.

Comment (dshoulders) [reply js/components/items-container.tsx]: Okay, so re-reading those docs it seems the "pagination" attribute should only apply to list values to turn pagination on/off. 
The "paginationSize" attribute should set the page size for all Service ObjectDataRequests and only list values where the "pagination" attribute has been set to true.

Comment (dshoulders) [reply js/components/items-container.tsx]: Now handling this in the getPageSize function.

File: js/components/items-container.tsx (hunk 1 above)
Comment (dshoulders) [reply js/components/items-container.tsx]: It would seem so. Fixed.

**Pull request merged**